README.md

```diff
@@ -1,114 +1,136 @@
 # Frontend Interview Preperation Guide
+
```

Comment:
@devlargs Can you please remove all the new lines and formatting changes? If we merge it with this, it would give merge conflict for other people. Please add only required changes in this PR.

Reply:
@hjaintech please check again. I removed the formatting and just added the answer. Thanks in advance.